eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -84,6 +84,22 @@ steps:
           -CancelPreviousBuilds $True
           -Base64EncodedAuthToken "$(azuresdk-azure-sdk-devops-build-queuing-pat)"
 
+    - task: PowerShell@2
+      displayName: Queue live-test template pipeline
+      condition: and(succeeded(), ne(variables['${{repo}}-template-tests-definition-id'], ''))
```

Comment:
While I'm Ok with this approach for now I would still like to move this list of definitions into the yml file instead of the pipeline queue time variables. Do we have an issue tracking that work?